apps/genetics/src/config.js

```diff
@@ -8,6 +8,7 @@ const config = {
   platformUrl: window.configPlatformUrl
     ? window.configPlatformUrl.replace(/\/$/, '')
     : 'https://platform.opentargets.org',
+  showTopBar: window.configShowTopBar ?? false,
```

Comment:
Is this line a fix for something overlooked in pull requests #1 and #2? Unless I'm mistaken, the top bar would never have shown up in the Genetics Portal without this line.

Reply:
Ah, never mind! It's a pull request to the v0.3.3/thehyve branch, where it is already implemented for the platform but not for the genetics portal.

Reply:
Exactly. sorry in the description I only mentioned the bug, but the original goal was to make sure TopBar shows correctly for OTG. Will update..